README.md

```diff
@@ -18,11 +17,21 @@ Once installed you can add a button to your pages by placing the *supersaas* sho
 ```
 [supersaas after=schedule_name label="Book Here!" image='http://cdn.supersaas.net/en/but/book_now_red.png']
 ```
+* A custom button example that opens a widget for `schedule_name` displayed as cards with menu at the top:
+```
+[supersaas after=schedule_name label="Pick a time!" view="card" menu_pos="top"]
```

Comment:
Looks good. But we should probably make `schedule` the first class citizen, since that's what 90% of the people need  and then make `after` the alternative, and motion that it can contain an URL instead of a schedule name only there.